charts/stable/ntfy/values.yaml

```diff
@@ -0,0 +1,50 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: binwiederhier/ntfy
+  # -- image tag
+  # @default -- chart.appVersion
+  tag:
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See more environment variables in the [ntfy documentation](https://ntfy.org/docs).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+
+# -- Configures command args to run the docker image in server mode
+# @default -- See values.yaml
+args:
+- serve
```

Comment:
```suggestion
  - serve
```

Reply:
Thanks for the lint. I guess old habits die hard. Resolved :)